docs/user/accumulo_datastore.rst

```diff
@@ -34,6 +34,18 @@ A script to bootstrap GeoMesa Accumulo on an Elastic Map Reduce cluster is provi
 Configuration
 -------------
 
+GeoMesa uses a site xml file to maintain system property configurations. This file can be found
+at ``conf/geomesa-site.xml`` of the Accumulo distribution. The default settings for GeoMesa are
+stored in ``conf/geomesa-default.xml``. Do not modify this file directly as it is never read,
+instead copy over the desired configurations into geomesa-site.xml.
+
+By default, system properties set through command line parameters will take precedent over this
```

Comment:
In general, this phrase should always be "take precedence," not "take precedent." See http://www.dailywritingtips.com/precedent-vs-precedence/